lib/utils/storage.ts

```diff
@@ -0,0 +1,61 @@
+import * as azure from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+const STORAGE_CONNECTION_STRING = process.env.AzureWebJobsStorage;
```

Comment:
only the main function should have access to the environment, utility functions should be "pure"

Reply:
ok